packages/block-library/src/columns/style.scss

```diff
@@ -14,6 +14,21 @@
 	}
 }
 
+// Stop Columns butting up against edge of viewport on smaller viewports. Remove
+// on larger viewports where width handles this. Specificity overide required to
+// overide generic `.entry-content *` rule
+.wp-block-columns {
+	&.wp-block-columns:not(.has-background) {
+		padding-left: $block-padding;
+		padding-right: $block-padding;
+
+		@include break-xlarge() {
+			padding-left: 0;
+			padding-right: 0;
+		}
```

Comment:
It looks like if it's an align-full block, that this padding applies, making it butt up against the window on the front-end:
<img width="1585" alt="Screen Shot 2020-03-02 at 4 02 18 PM" src="https://user-images.githubusercontent.com/967608/75722230-6b6bbe00-5c9f-11ea-8f1e-fafe5c674e4e.png">